src/workerd/api/global-scope.c++

```diff
@@ -439,6 +439,68 @@ kj::Promise<WorkerInterface::AlarmResult> ServiceWorkerGlobalScope::runAlarm(
   }
 }
 
+void ServiceWorkerGlobalScope::sendHibernatableWebSocketMessage(
+    kj::OneOf<kj::String, kj::Array<byte>> message,
+    Worker::Lock& lock, kj::Maybe<ExportedHandler&> exportedHandler) {
+  auto event = jsg::alloc<HibernatableWebSocketMessageEvent>();
+
+  KJ_IF_MAYBE(h, exportedHandler) {
+    KJ_IF_MAYBE(f, h->webSocketMessage) {
+      auto promise = (*f)(lock, kj::mv(message));
```

Comment:
`f` here is not very descriptive. A comment with a bit more detail would be helpful here.